library.sh

```diff
@@ -126,7 +126,7 @@ function subheader() {
 
 # Simple warning banner for logging purposes.
 function warning() {
-  make_banner "!" "$1"
```

Comment:
are we using the `make_banner` function somewhere else if not that can be removed

Reply:
The make_banner writes to a stdout. I wanted the warning to write to stderr, so when executed with capturing the output the warning will not influence the command.

The following will still work, but the deprecated warning will be captured in the build log.

```shell
deps="$(run_go_tool knative.dev/test-infra/buoy@latest buoy float ./go.mod "${buoyArgs[@]}")"
```

Reply:
Shouldn't you keep the original `make_banner "!" "$1"` but just pipe it to stderr then?

Reply:
Hmm, I'll test that. I don't know it that'll work...